docs/clients/cocoa/dsym.rst

```diff
@@ -132,7 +132,10 @@ Shell: `/bin/bash`
     export SENTRY_ORG=___ORG_NAME___
     export SENTRY_PROJECT=___PROJECT_NAME___
     export SENTRY_AUTH_TOKEN=YOUR_AUTH_TOKEN
-    sentry-cli upload-dsym
+    ERROR=$(sentry-cli upload-dsym 2>&1 >/dev/null)
+    if [ ! -z "$ERROR" ]; then
```

Comment:
Wouldn't it be more accurate to check returncode here instead of checking this string? It seems that this string will always be empty since you're piping all output to devnull.